### PR TITLE
Create stockfish wrapper to allow system agnostic creation of Stockfish object in stockfish_train

### PR DESCRIPTION
**Pull request by nashirj**

### CODE REVIEW

File: chess-AI/stockfish_train.py [hunk 1]
```diff
@@ -15,31 +19,22 @@ class StockfishTrain:
...
         """Sets the elo and depth for stockfish using the dashboard
 
         Attributes:
           dashboard: reference to the streamlit dashboard (this is temporary)
         """
-        
+
         # Set the elo and depth of the stockfish object
         self.stockfish.set_elo_rating(1000)
         self.stockfish.set_depth(3)
```
Comment (fshcat) [inline chess-AI/stockfish_train.py:34]: idk if this should go in a separate pr but elo and depth should be constructor arguments imo, i dont think set_params should exist

**Review by fshcat — APPROVED** [review]

File: chess-AI/stockfish_train.py (hunk 1 above)
Comment (nashirj) [reply chess-AI/stockfish_train.py:34]: Agree, I'll do it in this one
